model/testing/src/icon4py/model/testing/helpers.py

```diff
@@ -69,6 +78,27 @@ def allocate_data(backend, input_data):
     return input_data
 
 
+def match_marker(
+    markers: tuple[pytest.Mark | pytest.MarkDecorator, ...], backend: str, is_datatest: bool = False
+):
+    backend = None if backend == "embedded" else backend
```

Comment:
(1) backend after this stmt is `str | None` and later passed to `is_embedded`, but could also be passed to `is_dace` later (in case we have a marker that should be skipped for dace). But the `is_<backend>` functions take `Backend | None`.